src/cli.py

```diff
@@ -43,6 +43,27 @@ def cli():
 
     run(command + [name] + args)
 
+
+def add_test_user():
+    command = [
+        "docker", "run",
+        "-it",
+        "--network", "montagu_default"
+    ]
+
+    name = get_image_name("montagu-cli", versions.api)
+    args = ["add", "Test User", "test.user", "test.user@imperial.ac.uk", "password"]
+
+    run(command + [name] + args)
+
+    args = ["addRole", "test.user", "user"]
+
+    run(command + [name] + args)
+
+    args = ["addUserToGroup", "test.user", "ALL"]
```

Comment:
Can we add them as a reports reviewer too?

`args = ["addRole" "test.user" "reports-reviewer"]`

should be enough.  Then they get access to the reporting portal too